DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Commands/INoteCommands.cs

```diff
@@ -6,6 +6,6 @@ namespace GitHubExtension.Notes.WebApi.Commands
 {
     public interface INoteCommands
     {
-        Task AddNote(Note noteEntity);
+        Task AddOrUpdateNote(Note noteEntity);
```

Comment:
We should have only one note for collaborator from user and I decided not to call separate method for update a note, if one already exists. It is not single responsibility and decided for this simple case we can ignore it.
But now I think I should better have two methods CREATE and DELETE to avoid AddOrUpdate method. This way I will delete note, if it already exists for collaborator and then call create method. Also I do not have logic for deleting note as of now, so it can solve both moments.